include/smack/Naming.h

```diff
@@ -96,6 +96,7 @@ class Naming {
   static const std::vector<std::string> RUST_PANICS;
   static const std::string RUST_PANIC_ANNOTATION;
 
+  static const std::pair<std::string, std::string> INT_WRAP_FUNCTIONS;
```

Comment:
Would be less error prone to define two separate constants, e.g., `INT_WRAP_SIGNED_FUNCTION` and `INT_WRAP_UNSIGNED_FUNCTION`.

Reply:
Can you clarify it? Do we still need the getter function `getIntWrapFunc`?